homeassistant/components/verisure.py

```diff
@@ -52,6 +53,8 @@
         vol.Optional(CONF_GIID): cv.string,
         vol.Optional(CONF_HYDROMETERS, default=True): cv.boolean,
         vol.Optional(CONF_LOCKS, default=True): cv.boolean,
+        vol.Optional(CONF_DEFAULT_LOCK_CODE, default=None):
```

Comment:
We should not do a default and then match against `cv.string`. Since you use `.get` to get it from the dictionary, it will already return `None` instead of raise `KeyError` (which would happen when referring to a non existing key with `config['non-existing']`